app/models/founder.js

```diff
@@ -1,20 +1,19 @@
 import mongoose, { Schema } from 'mongoose';
 
 const FounderSchema = new Schema({
-    name: {
-      type: String,
-      required: true
-    },
-    email: {
-      type: String,
-      required: true
-    },
-    position: {
-        type: String,
-        required: true
-      }
-  });
+  name: {
+    type: String,
+    required: true,
+  },
+  email: {
+    type: String,
+    required: true,
+  },
+  position: {
```

Comment:
Can we rename this field to `title`?

(I understand you did not make this decision, but it would easy to edit this now)